package/yast2.spec

```diff
@@ -139,9 +142,21 @@ mkdir -p %{buildroot}%{yast_schemadir}/control/rnc
 mkdir -p %{buildroot}%{yast_schemadir}/autoyast/rnc
 mkdir -p %{buildroot}%{_sysconfdir}/YaST2
 
+rake install DESTDIR="%{buildroot}"
+
+# symlink to call yast in ncurses
+mkdir -p %{buildroot}%{_sbindir}
+ln -snf yast2 %{buildroot}%{_sbindir}/yast
+# backward compatibility
+mkdir -p %{buildroot}/sbin
+ln -snf yast2 %{buildroot}/sbin/yast2
+ln -snf yast2 %{buildroot}/sbin/yast
+ln -snf yast2.8 %{buildroot}/usr/share/man/man8/yast.8
 # symlink the yardoc duplicates, saves over 2MB in installed system
 # (the RPM package size is decreased just by few kilobytes
 # because of the compression)
+# remove local .rubocop
+rm %{buildroot}/usr/share/YaST2/lib/network/.rubocop.yml
```

Comment:
hmmm, sadly not, see link above. Only if I make regexp more precise. Or another option is to move these exception to global .rubocop.yml ( global in sense of /.y2rubocop in yast2 git repo ). instead of local file.